app/uk/gov/hmrc/trackingconsentfrontend/assets/test/domain/userPreference.spec.ts

```diff
@@ -49,8 +49,8 @@ describe('User Preference Factory', () => {
       testScope.fakeDatetime = 987654321
       userPreferenceFactory().userAcceptsAll()
       expectTrackingPreferenceToHaveBeenSetWith({
-        version: '2020-03-01',
-        dateSet: 987654321,
+        version: '2020.1',
+        datetime: 987654321,
```

Comment:
I thought we were going with datetimeSet?

Reply:
I'll update that now :) (and resolve conflicts)

Reply:
The conflicts were too painful so I re-implemented it against the latest master.